AndriyZhornyak/HMW3/HMW2.py

```diff
@@ -0,0 +1,3 @@
+a = "45689"
```

Comment:
look at this implementation
number = input("Enter a four digit number:")

# print(f'The product of numbers is: {int(number[0]) * int(number[1]) * int(number[2]) * int(number[3])}')

print(f'Reversed number: {number[::-1]}')

print(f'Sorted list: {list("".join(sorted(number)))}\n')